### DISCUSSION TITLE
Does DiskANN supports cosine metric type?

### DISCUSSION THREAD

**Discussion by rere950303** [milvus-io/milvus#30587] [Q&A and General discussion]
According to the official document(https://milvus.io/docs/disk_index.md#Limits), diskann does not support cosine metric type, but it seems to be working fine if i try it myself. Is there something wrong with the official document?

**Comment by yhmo** [comment]
DiskANN is available from v2.2, while COSINE metric is available from v2.3.0. The doc of DiskANN didn't mention COSINE because this doc is written in v2.2.
